### PR TITLE
MaroonX

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala [hunk 1]
```diff
@@ -14,8 +14,6 @@ import scala.annotation.tailrec
 sealed trait Overheads {
   // REL-2985 -> REL-2926: acquisitionOverhead and otherOverheadFraction are no longer used.
   def partnerOverheadFraction: Double // fpart
-  def acquisitionOverhead: Time       // acqover
```
Comment (cquiroz) [inline bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala:17]: This is a bit scary but these fields are not being used

File: bundle/edu.gemini.ags.client.impl/src/main/scala/edu/gemini/ags/client/impl/QueryArgs.scala [hunk 2]
```diff
@@ -58,6 +58,7 @@ object QueryArgs {
     case g: TexesBlueprint      => Right(Instrument.Nifs.id)      // REL-1062
     case g: VisitorBlueprint    => Right(Instrument.Niri.id)      // REL-1090
     case g: ZorroBlueprint      => Right(Instrument.Nifs.id)      // REL-3454: treat Zorro like NIFS.
+    case g: MaroonXBlueprint    => Right(Instrument.Nifs.id)
```
Comment (swalker2m) [inline bundle/edu.gemini.ags.client.impl/src/main/scala/edu/gemini/ags/client/impl/QueryArgs.scala:61]: Might add a comment here to indicate that this is intentional?

**Review by swalker2m — APPROVED** [review]
Looks great to me. :+1:

File: bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala (hunk 1 above)
Comment (sraaphorst) [reply bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala:17]: Nice to see this hot mess being cleaned up.

**Review by sraaphorst — APPROVED** [review]

**Pull request merged**